docs/outputs.rst

```diff
@@ -4,97 +4,151 @@ Outputs of tedana
+*_desc-TEDICAAcceptedZ_components.nii.gz   Z-normalized high-kappa ICA spatial component maps
+*_desc-TEDICA_comptable.tsv                TEDICA component table. A tab-delimited file with
+                                           summary metrics and inclusion/exclusion information
+                                           for each component from the ICA decomposition.
+========================================   =====================================================
 
 If ``verbose`` is set to True:
 
-======================    =====================================================
-Filename                  Content
-======================    =====================================================
-t2ss.nii                  Voxel-wise T2* estimates using ascending numbers
-                          of echoes, starting with 2.
-s0vs.nii                  Voxel-wise S0 estimates using ascending numbers
-                          of echoes, starting with 2.
-t2svG.nii                 Full T2* map/time series. The difference between
-                          the limited and full maps is that, for voxels
-                          affected by dropout where only one echo contains
-                          good data, the full map uses the single echo's
-                          value while the limited map has a NaN. Only used
-                          for optimal combination.
-s0vG.nii                  Full S0 map/time series. Only used for optimal
-                          combination.
-__meica_mix.1D            Mixing matrix (component time series) from ICA
-                          decomposition.
-hik_ts_e[echo].nii        High-Kappa time series for echo number ``echo``
-midk_ts_e[echo].nii       Mid-Kappa time series for echo number ``echo``
-lowk_ts_e[echo].nii       Low-Kappa time series for echo number ``echo``
-dn_ts_e[echo].nii         Denoised time series for echo number ``echo``
-======================    =====================================================
+=================================================   =====================================================
+Filename                                            Content
+=================================================   =====================================================
+*_desc-adaptiveGoodSignal_mask.nii.gz               Adaptive mask. Each voxel has value corresponding to
+                                                    number of echoes with good signal.
+*_desc-ascendingEstimates_t2s.nii.gz                Voxel-wise T2* estimates using ascending numbers
+                                                    of echoes, starting with 2.
+*_desc-ascendingEstimates_s0.nii.gz                 Voxel-wise S0 estimates using ascending numbers
+                                                    of echoes, starting with 2.
+*_desc-full_t2s.nii.gz                              Full T2* map/time series. The difference between
```

Comment:
I think we can just get rid of this `desc`. It should be clear that it's "full" in comparison to the `desc-limited` files above !

Reply:
I only don't like this because `full` is only used for optimal combination. It contains T2* estimates for voxels with only one good echo, which in my mind makes it less of a "full" map and more of a... liberal map? I'm not sure what to call it, but I think we should rename it. What about changing `limited` to nothing (dropping `desc`) and changing `full` to `liberal`?